extension/JuridicalPersion.StructureDefinition.json

```diff
@@ -0,0 +1,53 @@
+{
+  "resourceType": "StructureDefinition",
+  "url": "http://bbmri.de/fhir/StructureDefinition/JuridicalPersion",
```

Comment:
The domain name we like to use is fhir.bbmri.de. It's important that this is a subdomain because we like to have our own server with redirects there. Also please use HTTPS.